include/daisykitsdk/models/facial_landmark_estimator.h

```diff
@@ -16,51 +16,44 @@
 #define DAISYKIT_MODELS_FACIAL_LANDMARK_ESTIMATOR_H_
 
 #include <daisykitsdk/common/types.h>
-
+#include <daisykitsdk/models/base_model.h>
 #include <opencv2/opencv.hpp>
-#ifdef __ANDROID__
-#include <android/asset_manager_jni.h>
-#endif
 
-// Ncnn
-#include <benchmark.h>
-#include <cpu.h>
-#include <datareader.h>
-#include <gpu.h>
-#include <net.h>
-#include <platform.h>
 namespace daisykit {
 namespace models {
 
-class FacialLandmarkEstimator {
+class FacialLandmarkEstimator
+    : public BaseModel<cv::Mat, std::vector<daisykit::types::Keypoint>> {
  public:
+  FacialLandmarkEstimator(const char* param_buffer,
+                          const unsigned char* weight_buffer,
+                          int input_width = 112, int input_height = 112);
+
+  // Will be deleted when IO module is supported. Keep for old compatibility.
   FacialLandmarkEstimator(const std::string& param_file,
-                          const std::string& weight_file);
-  void LoadModel(const std::string& param_file, const std::string& weight_file);
-#ifdef __ANDROID__
-  FacialLandmarkEstimator(AAssetManager* mgr, const std::string& param_file,
-                          const std::string& weight_file);
-  void LoadModel(AAssetManager* mgr, const std::string& param_file,
-                 const std::string& weight_file);
-#endif
-  // Detect keypoints for single object
```

Comment:
We use `///` for the documentation. I think we should put a dot (`.`) to each documented comments.
Example:

```
// Detect keypoints for single object
std::vector<daisykit::types::Keypoint> Detect(cv::Mat& image,
```

New code

```
/// Detect keypoints for single object.
std::vector<daisykit::types::Keypoint> Detect(cv::Mat& image,
```